ep/build-velox/src/build_velox.sh

```diff
@@ -282,9 +282,11 @@ function compile_arrow_java_module() {
     ARROW_INSTALL_DIR="${ARROW_HOME}/../../install"
 
     pushd $ARROW_HOME/java
-    
-    mvn clean install -pl maven/module-info-compiler-maven-plugin -am \
-          -Dmaven.test.skip -Drat.skip -Dmaven.gitcommitid.skip -Dcheckstyle.skip
+    # Because arrow-bom module need the -DprocessAllModules
+    mvn versions:set -DnewVersion=15.0.0-gluten -DprocessAllModules
+   
+    mvn clean install -am \
+          -DskipTests -Drat.skip -Dmaven.gitcommitid.skip -Dcheckstyle.skip
```

Comment:
do we really need compile whole project? it's a time consuming command...
I wonder which depends module can not be compiled by `-am` options?

Reply:
arrow-bom module includes many modules, so we need to compile whole project